### PR TITLE
CI Visibility: document Intelligent Test Runner for pytest

### PR DESCRIPTION
**Pull request by romainkomorndatadog**
### What does this PR do?
Adds documentation for the Intelligent Test Runner for pytest under the Python section.

### Motivation
Intelligent Test Runner support for pytest is being announced as a beta at Dash, and will be available in the Python `ddtrace` (in version `1.18.0`).

---

### Reviewer checklist
- [x] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [x] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/continuous_integration/intelligent_test_runner/python.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+  * From `Python>=3.7`.
+  * Requires `coverage>=5.5`.
+
+## Setup
+
+Prior to setting up Intelligent Test Runner, set up [Test Visibility for Python][1]. If you are reporting data through the Agent, use v6.40+/v7.40+.
+
+To enable Intelligent Test Runner, set the following environment variables:
+
+`DD_APPLICATION_KEY` (Required)
```
Comment (gnufede) [inline content/en/continuous_integration/intelligent_test_runner/python.md:30]: nit: They are equivalent but I think we're moving to `DD_APP_KEY` as used in the code below

File: content/en/continuous_integration/intelligent_test_runner/python.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+## Disabling the Intelligent Test Runner
+
+The Intelligent Test Runner can be disabled at runtime, regardless of the settings applied in the UI activation section above, by setting the `DD_CIVISIBILITY_ITR_DISABLED` environment variable to `true`.
+
+`DD_CIVISIBILITY_ITR_DISABLED` (Optional)
+: Disables the Intelligent Test Runner from running.<br/>
+**Default**: `false`
+
+## Further Reading
```
Comment (brett0000FF) [inline content/en/continuous_integration/intelligent_test_runner/python.md:53]:
```suggestion
## Further reading
```

File: content/en/continuous_integration/intelligent_test_runner/python.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+further_reading:
+    - link: "/continuous_integration/tests"
+      tag: "Documentation"
+      text: "Explore Test Results and Performance"
+    - link: "/continuous_integration/troubleshooting/"
+      tag: "Documentation"
+      text: "Troubleshooting CI"
+---
+
+{{< callout url="#" btn_hidden="true" >}}Intelligent Test Runner for Python (using `pytest`) in beta.{{< /callout >}}
```
Comment (brett0000FF) [inline content/en/continuous_integration/intelligent_test_runner/python.md:13]: Did you intend to put a sign up form or some other URL in this callout?

**Review by brett0000FF — CHANGES_REQUESTED** [review]
Great content! I just had a question about the beta banner before approving. Did you want to include a link in there? The heading says "Join the Beta", but there isn't a link to a sign up form like I typically see.

Thanks!

File: content/en/continuous_integration/intelligent_test_runner/python.md (hunk 3 above)
Comment (romainkomorndatadog) [reply content/en/continuous_integration/intelligent_test_runner/python.md:13]: That is a great question. I borrowed it from our Java page ( https://docs.datadoghq.com/continuous_integration/intelligent_test_runner/java/?tab=gradle ), and I don't actually know if we have a form for it, but I'll find out.

Comment (romainkomorndatadog) [reply content/en/continuous_integration/intelligent_test_runner/python.md]: @brett0000FF , I removed the header (so there's no more callout), and also removed the backticks around `pytest` that weren't rendering. Feel free to resolve this conversation if you're happy with the change. :)

Comment (brett0000FF) [reply content/en/continuous_integration/intelligent_test_runner/python.md]: @romainkomorndatadog , great--looks good! Let me know if you need me to merge this for you. Oh, for future reference, you should be able to wrap text like pytest in <code></code> to get the monospace formatting within callouts.

**Review by brett0000FF — APPROVED** [review]

**Review by ericlaz — APPROVED** [review]

**Comment by romainkomorndatadog** [comment]
Moving this back to WIP since there're some significant changes.

File: content/en/continuous_integration/intelligent_test_runner/python.md [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+{{< /tabs >}}
+
+
+## Running tests with the Intelligent Test Runner enabled
+
+### Environment variable
+<div class="alert alert-info">Setting DD_CIVISIBILITY_AGENTLESS_ENABLED to true is required while the Intelligent Test Runner support for pytest is in beta. </div>
+
+`DD_CIVISIBILITY_AGENTLESS_ENABLED` (Optional)
+: Run the Intelligent Test Runner in agentless mode (also applies to test visibility).
```
Comment (brett0000FF) [inline content/en/continuous_integration/intelligent_test_runner/python.md:71]:
```suggestion
: Run the Intelligent Test Runner in Agentless mode (also applies to test visibility).
```

**Review by brett0000FF — APPROVED** [review]
Looks good!

File: content/en/continuous_integration/intelligent_test_runner/python.md (hunk 4 above)
Comment (romainkomorndatadog) [reply content/en/continuous_integration/intelligent_test_runner/python.md]: Resolving because I removed that line.

**Review by juan-fernandez — COMMENTED** [review]
other than a broken link the js section, LGTM

File: content/en/continuous_integration/intelligent_test_runner/javascript.md [hunk 5]
```diff
@@ -69,7 +66,6 @@ NODE_OPTIONS="-r dd-trace/ci/init" DD_ENV=ci DD_SERVICE=my-javascript-app DD_CIV
 
 {{< /tabs >}}
 
-
 ### Cypress setup
 
 For Intelligent Test Runner for Cypress to work, you must instrument your web application with code coverage. You can read more about enabling code coverage in the [Cypress documentation][4]. To check that you've successfully enabled code coverage, navigate to your web app with Cypress and check the global variable `window.__coverage__`. This is what `dd-trace` uses to collect code coverage for Intelligent Test Runner.
```
[the remark this answers is not in the document]
Comment (romainkomorndatadog) [reply content/en/continuous_integration/intelligent_test_runner/javascript.md]: Fixed.

**Review by cswatt — APPROVED** [review]

**Pull request merged**